tst/working/quick/sl3.tst

```diff
@@ -1,11 +0,0 @@
-gap> G:=Group([
->   [ [ Z(25)^20, Z(25)^8, Z(25)^23 ],
->     [ Z(25)^20, Z(25), Z(25)^16 ],
->     [ Z(25)^21, Z(25)^23, Z(25)^13 ] ],
->   [ [ Z(25)^16, Z(25)^0, Z(25)^4 ],
->     [ Z(25)^14, Z(25)^4, Z(25)^19 ],
->     [ Z(25)^23, Z(25)^17, Z(25)^12 ] ]
-> ]);;
-
-#
-gap> ri := RECOG.TestGroup(G, false, 152334000000);;
```

Comment:
This test should be kept here, in the "quick" section, so its coverage is not reduced. If you dislike the duplicate, remove the copy in the "slow" test suite.

Reply:
I have instead made it so more of the tests from the `ClassicalNatural.tst` are now run in the `quick/` folder. I used `GL`s though I think that's no problem. For `GL`s we use the determinant homomorphism, then we recognize the `SL` in its kernel.